core/modules/layout/layout.theme.inc

```diff
@@ -137,6 +137,19 @@ function theme_layout_conditions($variables) {
 }
 
 /**
+ * Output region content wrapped by tag specified in layout UI.
+ */
+function theme_layout_region_inner($variables) {
+  $tag = $variables['tag'];
+  $classes = $variables['classes'];
+
+  if ($tag != '') {
+    return '<' . $tag . ' class="' . $classes .'">' . implode('', $variables['blocks']) . '</' . $tag . '>';
```

Comment:
For flexibility and consistency with block and layout templates, $attributes should also be passed in and added to the wrapper.